spring-boot-modules/spring-boot-data-2/src/test/java/com/baeldung/boot/customRepository/CustomRepositoryUnitTest.java

```diff
@@ -0,0 +1,28 @@
+package com.baeldung.boot.customRepository;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import com.baeldung.boot.customRepository.model.User;
+import com.baeldung.boot.customRepository.repository.UserRepository;
+
+@SpringBootTest(classes = CustomRepositoryApplication.class)
+public class CustomRepositoryUnitTest {
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Test
+    public void givenCustomRepository_whenInvokeCustomFindMethod_thenEntityIsFound() {
```

Comment:
junit5 tests could be default package scope
```suggestion
    void givenCustomRepository_whenInvokeCustomFindMethod_thenEntityIsFound() {
```